en/Greedy Algorithms/Fractional Knapsack.md

```diff
@@ -0,0 +1,52 @@
+# Knapsack Problem (Greedy algorithm)
+
+#### Problem Statement
+
+Given a set of items, each with weight and a value, determine the number of each item included in a collection so that the total weight is less than or equal to the given limit and the total value is as large as possible.
+
+##### Greedy method will always provide an optimal solution with fractional knapsack problem.
+
+#### Time Complexity
+
+O(log n) Worst Case     
```

Comment:
Why? Looks wrong to me: In the worst case, you will have to pack all `n` items, so this should be at least `O(n)`?